### PR TITLE
Add motivation paragraph and comments from Luke

### PR DESCRIPTION
**Pull request by joelostblom**

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -78,8 +78,44 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
 
 ## Story of the project
 
-We are a group of graduate students who connected through the [UofT Coders](http://uoftcoders.github.io/) club at the University of Toronto. We created and ran a fourth-year course in the Department of Ecology and Evolutionary Biology titled “Theoretical Ecology and Reproducible Quantitative Methods in R.” We modelled the structure and portions of the course content on “Reproducible Quantitative Methods” a course created by Dr. Christie Bahlai [ ], modifying the lesson content to include more theoretical ecology topics but maintaining the focus on reproducibility and computational skills. Our teaching team was six graduate students from several different fields (Physiology, Biomedical Engineering, Physics, Psychology, and Nutritional Science); we divided course topics among each instructor to develop and deliver individual lessons and assignments.
+While there are many excellent open source libraries for quantitative data
+analysis, the use of subpar analysis tools (such as spreadsheet software) is
```
Comment (mbonsma) [inline paper.md]: Do we want to make a value judgment on spreadsheet software? Maybe "inefficient and unreproducible"

File: paper.md [hunk 2]
```diff
@@ -78,8 +78,44 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+still prevalent among graduate students. Part of this issue stems from lack of
+awareness, and part from that students are often required to learn about new
+tools in isolation and outside excessive amounts of time already spent on their
```
Comment (mbonsma) [inline paper.md]: `in isolation and on top of their main research activities` (shorter)

**Comment by mbonsma** [comment]
I'm not 100% sure that this is the best place for this type of justification - does this fall under 'story of the project' or something more like the introduction?

File: paper.md [hunk 3]
```diff
@@ -78,8 +78,44 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+Like-minded peers could provide the needed support structure to uphold
+motivation, but are often few and far between, especially in fields without a
+strong quantitative culture. To remedy these issues, we launched the student
+group [UofT Coders](http://uoftcoders.github.io/) which teaches how to use code
+for research through skill sharing, co-working and community building in a
+friendly environment.
 
-Following a successful term with a class of eight students, we delivered the course a second time to a class of twenty-six students with a teaching team of four graduate students (Physics, Ecology and Evolutionary Biology, Psychology, and Cell and Systems Biology). We modified our lecture material to include more generally applicable statistical concepts and fewer theoretical ecology concepts, and renamed the course "Quantitative Methods in R for Biology."
+After receiving overwhelmingly positive feedback on content and teaching style,
+we sought to formally disseminate our experiences through the university
```
Comment (mbonsma) [inline paper.md]: Can we use `share` instead of `disseminate`? Disseminate is so... seminal.

File: paper.md [hunk 4]
```diff
@@ -78,8 +78,44 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+curriculum. We designed a course on open, reproducible, data analysis, which we
```
Comment (mbonsma) [inline paper.md]: No comma after 'reproducible'

File: paper.md [hunk 5]
```diff
@@ -78,8 +78,44 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
...
+Methods"](https://cbahlai.github.io/rqm-template/) a course created by Dr.
+Christie Bahlai, modifying the lesson content to include additional theoretical
+ecology topics but maintaining the focus on reproducibility and computational
+skills. Our teaching team consisted of six graduate students from several
+different fields (Physiology, Biomedical Engineering, Physics, Psychology, and
+Nutritional Science); we divided course topics among each instructor to develop
+and deliver individual lessons and assignments.
+
+Following a successful pilot term with a class of eight students, the course
+was incorporated into the long-term curriculum and delivered a second time to a
```
Comment (mbonsma) [inline paper.md:112]: Yaaassss 'long-term curriculum' :smile:

**Review by mbonsma — CHANGES_REQUESTED** [review]
Just a few small things!

File: paper.md (hunk 1 above)
Comment (joelostblom) [reply paper.md]: Haha, I went back and forth on this like 5 times, between 'inefficient' or 'subpar'. For the reasons you mentioned. Including reproducibility is good, will change in a bit.

**Review by mbonsma — APPROVED** [review]

**Pull request merged**